### PR TITLE
Update supported python and django versions

### LINKED ISSUE #91: Add supported Django versions to README
Similar to: https://github.com/tomchristie/django-rest-framework#requirements

### LINKED ISSUE #121: Support Django >= 2.2
Django 2.2 works fine, it's being used in some of Vinta's projects.  
But we're not sure about Django 3  

Please do: 
- [ ] review https://github.com/vintasoftware/django-templated-email/pull/119, see if solves this issue
- [ ] add support to all Django >= 2.2 (see image below). 
- [ ] drop support to all Django < 2.2 and Python 2 (those are support as of today, see tox.ini)
- [ ] test all Django >= 2.2 manually

image

### PR DESCRIPTION
**Pull request by tuliolages**
Closes #121 
Closes #91

Updates Python and Django as specified in the issue #121

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,48 +5,42 @@ matix:
...
     env:
-      - TOX_ENV=py37-django30
+      - TOX_ENV=py37-django32
   - python: 3.8
     env:
-      - TOX_ENV=py38-django30
+      - TOX_ENV=py38-django32
+  - python: 3.9
+    env:
+      - TOX_ENV=py39-django32
```
Comment (fjsj) [inline .travis.yml:43]: To avoid manually specifying all versions, we can try: https://github.com/tox-dev/tox-travis#usage

Comment (fjsj) [reply .travis.yml:43]: Note this should be installed only on `.travis.yml`, so you can add it to the `install` block below

Comment (tuliolages) [reply .travis.yml]: @fjsj Thanks for the tip, it worked

**Comment by fjsj** [comment]
@tuliolages could you please do the following?
- update README Python badge to the current supported Python versions *check the bottom of the README code)
- add a Requirements section describing the supported Python and Django versions, like https://github.com/encode/django-rest-framework#requirements
- remove the requirement six and remove six imports from code. On `isinstance(klass_path, six.string_types)`, you can replace to `isinstance(klass_path, str)`
- remove line `from __future__ import unicode_literals` from `templated_email/migrations/0001_initial.py`

**Review by fjsj — CHANGES_REQUESTED** [review]
Please check comment above ^

**Review by fjsj — APPROVED** [review]
LGTM. Merging. Thanks!

**Pull request merged**